source/coding-the-legislation/35_periods.md

```diff
@@ -42,12 +38,48 @@ This [YAML test](writing_yaml_tests.md) on `income_tax` evolution over time show
       2015-01: 416.6667 # The income tax rate changes in 2015
       2016-01: 416.6667
       2017-01: 0 # The salary is not set for this period and defaults to 0
+```
+
+This syntax could also be used in Python.  
+This example sets 3 years salaries for two persons and calculates how much they get for one month:
 
+```py
+# Each person earned 60,000 between 2014 and 2016
+simulation.set_input('salary', 'year:2014:3', [60000.0, 60000.0])
+
+one_month_salaries = simulation.calculate('salary', 'month:2014-01:1')
+print(one_month_salaries)  # prints [1666.6666 1666.6666]
+```
+
+## Manipulating Period objects in Python
+
+To create a `Period`, you can use a simplified syntax. Here is an example for a period of one year covering `2015`:
+
+```py
+from openfisca_core import periods
+
+period_2015 = periods.period('2015')
+```
+
+> Internally, periods are stored as:
+> - a start `Instant`
+> - a unit (`DAY`, `MONTH`, `YEAR`)
+> - a quantity of units.
+
+The previous example could also be defined as:
+
+```py
+from openfisca_core import periods
+
+period_2015 = periods.Period(('month', periods.Instant((2015, 1, 1)), 12))
 ```
 
 ## Periods in variable definition
 
 ```py
+...
+from openfisca_core.periods import MONTH  # openfisca_core is the architecture of OpenFisca
```

Comment:
Not sure about the meaning of "# openfisca_core is the architecture of OpenFisca".
Not sure either about why we are importing that here. Why not import `Person`, `Variable` too? Variables files usually don't import manually all the toolbox, and I think it's misleading to do it in the doc.